js/islandora_internet_archive_bookreader.js

```diff
@@ -35,6 +35,10 @@ Drupal.settings.islandoraInternetArchiveBookReader_jQuery = jQuery.noConflict(tr
         if ($.browser.mobile && settings.islandoraInternetArchiveBookReader.mobilize) {
           bookReader.goFullScreen();
         }
+        // Add class 'aria-label' for accessibility support for screen readers.
+        $('.ui-slider-handle').each(function(e){
```

Comment:
Does this work if .ui-slider-handle is undefined? For a single page book do we get this slider?

Reply:
Just tested, the slider does appear for a single page book and the aria-label set here is present.